lib/code_fund/schemas/budgeted_campaign.ex

```diff
@@ -0,0 +1,30 @@
+defmodule CodeFund.Schema.BudgetedCampaign do
+  @moduledoc """
+  This schema is for a postgres materialized view
+  """
+
+  use CodeFundWeb, :schema
+
+  @primary_key false
+  @foreign_key_type :binary_id
+  schema "budgeted_campaigns" do
+    belongs_to(:campaign, CodeFund.Schema.Campaign)
+    belongs_to(:user, CodeFund.Schema.User)
```

Comment:
Do we need all of these relationships (aside from User)? Looks like we just query this on campaign only.

Reply:
Probably not. I will remove them.